src/py/flwr/server/superlink/state/in_memory_state.py

```diff
@@ -251,6 +254,36 @@ def create_run(self, fab_id: str, fab_version: str) -> int:
         log(ERROR, "Unexpected run creation failure.")
         return 0
 
+    def store_server_public_private_key(
+        self, public_key: bytes, private_key: bytes
+    ) -> None:
+        """Store `server_public_key` and `server_private_key` in state."""
+        with self.lock:
+            self.server_private_key = private_key
+            self.server_public_key = public_key
+
+    def get_server_private_key(self) -> bytes:
+        """Retrieve `server_private_key` in urlsafe bytes."""
+        return self.server_private_key if self.server_private_key is not None else b""
+
+    def get_server_public_key(self) -> bytes:
+        """Retrieve `server_public_key` in urlsafe bytes."""
+        return self.server_public_key if self.server_public_key is not None else b""
```

Comment:
Does it make sense to return an empty `str` here? Or should we make the return type `Optional[bytes]` and return `None`?

Reply:
Changed state and its implementation to return optional bytes.